### PR TITLE
[rocm-opencl-runtime] Update to 3.5.0

### PR DESCRIPTION
**Pull request by tpkessler**
We now build the OpenCL runtime with ROCclr. To my surprise, this new release also fixes the OpenCL bugs I had.

### CODE REVIEW

File: rocm-opencl-runtime/PKGBUILD [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
 provides=("$pkgname" 'opencl-driver')
-source=("$url/archive/roc-$pkgver.tar.gz"
-        "$_opencl_icd_loader_repo/archive/$_opencl_icd_loader_commit.tar.gz"
-        'install_vendor_file.patch')
-sha256sums=('ac6999f1a491ab066286c2bd6adf50f08f831286f56e267879f9f7eced22f98e'
-            '0c14bf890bd198ef5a814b5b7ed57b69e890b0c0a1bcfba8fdad996fa1a97fc7'
-            'b83de5ea8ae889664ce2725f90c5db8c1c9e98839d75c7743b355d16435dccee')
+source=("$pkgname-$pkgver.tar.gz::$url/archive/roc-$pkgver.tar.gz")
+        # 'install_vendor_file.patch'
+        # 'remove-clinfo.patch'
```
Comment (acxz) [inline rocm-opencl-runtime/PKGBUILD:15]: If we don't need the patches/if they don't work anymore lets go ahead and delete the patches and their associated lines of code in the PKGBUILD.

Comment (tpkessler) [reply rocm-opencl-runtime/PKGBUILD]: Done.

Comment (acxz) [reply rocm-opencl-runtime/PKGBUILD]: Seems like one of the patch files is still there?

Comment (tpkessler) [reply rocm-opencl-runtime/PKGBUILD]: Ups. I've removed it.

**Pull request merged**